training/controllers/articles.py

```diff
@@ -0,0 +1,113 @@
+from flask import Blueprint, request, jsonify, g
+
+from training.controllers.function_decorators import login_required
+from training.extensions import db
+from training.models.articles import Article, ArticleSchema
+from training.models.users import User
+
+bp = Blueprint('article', __name__, url_prefix='/articles')
+
+
+@bp.route('/register', methods=['POST'])
+@login_required
+def create_article():
+    if not request.json or "title" not in request.json or "body" not in request.json:
+        return jsonify({'Error': 'Bad request, "title" and "body" is required!'}), 400
+
+    if len(request.json['title']) > 50 or len(request.json['body']) > 1000:
+        return jsonify({'Error': 'Bad request, max length for "title" is 50 and max length for "body" is 1000.'}), 400
```

Comment:
I think you have already used marshmellow validators right? We should use them here as well

Reply:
Yes, that is true, i will add it!